plugin.json

```diff
@@ -100,6 +100,13 @@
         "type": "bool",
         "help_text": "Display subscription name in post when a subscription posts to a channel",
         "default": false
+      },
+      {
+        "key": "EnableWebhookEventLogging",
+        "display_name": "Enable Webhook Event Logging:",
```

Comment:
Seems weird how we have colons for some of the `display_name` but not all. @sibasankarnayak Do you mind adding colons to the rest of the settings as well? Also the periods at the end of the `help_text`